index.js

```diff
@@ -460,18 +475,20 @@ module.exports = function (log, indexesPath) {
       loadTypedArrayFile(
         index.filepath,
         Uint32Array,
-        (err, { seq, tarr, count }) => {
+        (err, { version, seq, count, tarr }) => {
           // FIXME: handle error
+          index.version = version
           index.seq = seq
-          index.tarr = tarr
           index.count = count
+          index.tarr = tarr
           index.lazy = false
           cb()
         }
       )
     } else {
-      loadBitsetFile(index.filepath, (err, { seq, bitset }) => {
+      loadBitsetFile(index.filepath, (err, { version, seq, bitset }) => {
```

Comment:
and count should be here as well

Reply:
What's the reason? `count` was already available before this PR, and `count` was not used in this line of code